### ISSUE TITLE
Would it be possible to add the Bloom model and other multilanguage models?

### ISSUE THREAD

**Issue by asterbini** [ollama/ollama#4406]
It would be nice, for us non-English people, to have access to the some good multilanguage LLMs.
Bloom comes to mind, but other would be very useful.

https://huggingface.co/blog/bloom

**Comment by thinkverse** [comment]
Yes, Bloom's `BloomForCausalLM` architecture is supported by llama.cpp. There are 2 1B Bloom models on Ollama already which were added by a community member:

- https://ollama.com/gouranshitera/bloom-1b1
- https://ollama.com/gouranshitera/bloomz-1b7

You can convert models yourself by using the [import guide](https://github.com/ollama/ollama/blob/main/docs/import.md), make sure you use [convert-hf-to-gguf.py](https://github.com/ggerganov/llama.cpp/blob/master/convert-hf-to-gguf.py#L667) and not `convert.py` during the [convert the model](https://github.com/ollama/ollama/blob/main/docs/import.md#convert-the-model) step, and don't use Python 3.12 or later that's not supported yet. I've had success converting models using Python 3.11.
